packages/fx-core/src/plugins/resource/appstudio/plugin.ts

```diff
@@ -1137,28 +1151,8 @@ export class AppStudioPluginImpl {
 
   private getSelectedPlugins(ctx: PluginContext): Result<Plugin[], FxError> {
     const azureSettings = ctx.projectSettings?.solutionSettings as AzureSolutionSettings;
-
-    const plugins = new Map<string, Plugin>();
-    for (const k in ResourcePlugins) {
-      const plugin = Container.get<Plugin>(k);
-      if (plugin) {
-        plugins.set(plugin.name, plugin);
-      }
-    }
-
-    const results: Plugin[] = [];
-    for (const name of azureSettings.activeResourcePlugins) {
-      const plugin = plugins.get(name);
-      if (!plugin) {
-        return err(
-          AppStudioResultFactory.UserError(
-            AppStudioError.PluginNotFound.name,
-            AppStudioError.PluginNotFound.message(name)
-          )
-        );
-      }
-      results.push(plugin);
-    }
-    return ok(results);
+    const plugins = getActivatedResourcePlugins(azureSettings);
+    azureSettings.activeResourcePlugins = plugins.map((p) => p.name);
```

Comment:
seems the assignment to azureSettings is not used anymore.

getSelectedPlugins equals getActivatedResourcePlugins here.

Reply:
Yes, just copy from solution. This method can be deleted